packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -176,24 +208,70 @@
 		overflow-y: auto;
 	}
 
-	h2 {
+	.heading {
+		position: relative;
+
+		width: 100%;
+
+		display: grid;
+		align-items: center;
+		grid-template-columns: 1fr auto;
+		gap: 0.75rem;
+
+		padding: 1rem 0.75rem;
+	}
+
+	.h2 {
 		text-transform: uppercase;
 		font-size: 1.4rem !important;
 		font-weight: 400;
-		margin: 0 0 1rem 0 !important;
-		padding: 0 0 0 0.6rem;
+		margin: 0 !important;
+		padding: 0 !important;
 		color: var(--sk-text-3);
+		text-align: start;
+	}
+
+	.heading :global(svg) {
+		display: none;
+
+		transform: translateY(-1px);
+	}
+
+	.expand-icon :global(svg) {
+		transition: transform 0.4s var(--quint-out);
+		transform-origin: center;
+	}
+
+	.expand-icon.inverted :global(svg) {
+		transform: rotate3d(0, 0, 1, 180deg);
 	}
 
 	ul {
-		list-style: none;
+		list-style: none !important;
+		margin-left: 0 !important;
+	}
+
+	li {
+		margin: 0.2rem !important;
+	}
+
+	li::before {
+		content: none !important;
+	}
```

Comment:
we're using a lot of `!important` here - is that really necessary? what styles are we trying to override?

Reply:
Its inside `class="text"`. !important is needed to override those

Reply:
Opened this PR for your review https://github.com/sveltejs/site-kit/pull/167